libos/include/libos_ipc.h

```diff
@@ -282,6 +282,7 @@ struct libos_ipc_posix_lock {
     uint64_t start;
     uint64_t end;
     IDTYPE pid;
+    uint64_t handle_id;
```

Comment:
Might be a stupid question, but are we using IPC locks for `flock`? In which testcase would we hit this?

Reply:
Yes, When we use `fork` then in the child process the `g_process_ipc_ids.leader_vmid != 0`.

Reply:
Thanks for the explanation!